src/stylesheets/main.scss

```diff
@@ -87,6 +87,13 @@ $mq-breakpoint-widescreen: 1200px;
     max-width: 38em;
   }
 
+  img {
+    box-sizing: border-box;
+    width: 100%;
```

Comment:
Should this be max-width: 100%; to avoid scaling up images that are smaller than the container? I think it also needs `height: auto;`

Reply:
I have added the `height: auto;` however using `max-width` actually breaks the image out of the faux-column when the <p> tags around it are removed.

So really it needs to be width: 100%; max-width: 38em;

Thoughts?

Reply:
Hmmm. The issue is that `width: 100%` will scale up anything smaller than the container. Suggest adding a small image and testing with that? Worth noting that in the prototype we had an additional feature that linked images to allow you to open them in a new window / tab – we might want to copy that functionality in which case we could also look at adding a wrapper if we need to.